packages/filesystem/src/browser/file-dialog/file-dialog.ts

```diff
@@ -158,7 +165,7 @@ export abstract class FileDialog<T> extends AbstractDialog<T> {
     }
 
     protected createLocationListRenderer(): LocationListRenderer {
-        return new LocationListRenderer(this.model);
+        return new LocationListRenderer(this.model, undefined, this.fileService);
```

Comment:
The calls here are a little odd. I think the clarity of the flow would be improved if `createLocationListRenderer()` took an optional `host` argument to be passed into the `new LocationListRenderer()` call, so that it's clear what the `undefined` corresponds to. Then it might make sense just to create the `host` around line 155 rather than waiting for the `LocationListRenderer` to create its own host.

Reply:
I like that idea, I've made that change 👌